### PR TITLE
Adding new CLI parameters to support for proxy config

### PR DESCRIPTION
**Pull request by pasankav**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

File: src/connectedk8s/azext_connectedk8s/custom.py [hunk 1]
```diff
@@ -250,6 +259,9 @@ def create_connectedk8s(cmd, client, resource_group_name, cluster_name, location
                         "--set", "global.resourceName={}".format(cluster_name),
                         "--set", "global.location={}".format(location),
                         "--set", "global.tenantId={}".format(onboarding_tenant_id),
+                        "--set", "global.httpsProxy={}".format(https_proxy),
+                        "--set", "global.httpProxy={}".format(http_proxy),
+                        "--set", "global.noProxy={}".format(no_proxy),
```
Comment (fengzhou-msft) [inline src/connectedk8s/azext_connectedk8s/custom.py:264]: Just to confirm, when `https_proxy` is None, the value is `global.httpsProxy=""` and will be processed correctly by helm?

Comment (pasankav) [reply src/connectedk8s/azext_connectedk8s/custom.py:264]: We have handled it in helm chart, to set these values only if they exists.

**Comment by yonzhan** [comment]
connectedk8s

**Comment by fengzhou-msft** [comment]
@pasankav I just realized that you need to bump the version in `setup.py` too. Please submit another PR.

**Pull request merged**